CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## How to work on pylight.org website locally (without heroku)
+
+This way is more Django oriented, and closer to how most apps are developed.
+
+1. Make sure you've python 3.6.x installed on your machine 
+2. Create virtualenv using python3.6 and activate it.
+3. `pip install -r requirements.txt`
+4. `source .env` (this includes some required environment variables)
```

Comment:
Info about `SECRET_KEY` env var would be helpful at this stage - either set it in .env file or explicitly instruct the developer to set it